bodhi/server/mail.py

```diff
@@ -241,102 +242,31 @@
-%(notes)s%(changelog)s%(references)s
-This update can be installed with the "dnf" update programs.  Use
-su -c 'dnf%(yum_repository)s upgrade %(name)s' at the command line.
-For more information, refer to the dnf documentation available at
-http://dnf.readthedocs.io/en/latest/command_ref.html#upgrade-command-label
-
-All packages are signed with the Fedora EPEL GPG key. More details on the
-GPG keys used by the Fedora Project can be found at
-https://fedoraproject.org/keys
---------------------------------------------------------------------------------
-"""
-
-# message template for legacy systems, still pointing users to use "yum"
-fedora_epel_legacy_errata_template = u"""\
---------------------------------------------------------------------------------
-Fedora EPEL%(testing)s Update Notification
-%(updateid)s
-%(date)s
---------------------------------------------------------------------------------
-
-Name        : %(name)s
-Product     : %(product)s
-Version     : %(version)s
-Release     : %(release)s
-URL         : %(url)s
-Summary     : %(summary)s
-Description :
-%(description)s
-
---------------------------------------------------------------------------------
-%(notes)s%(changelog)s%(references)s
-This update can be installed with the "yum" update programs.  Use
-su -c 'yum%(yum_repository)s update %(name)s' at the command line.
-For more information, refer to "YUM", available at
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7\
-/html/System_Administrators_Guide/ch-yum.html
-
-All packages are signed with the Fedora EPEL GPG key.  More details on the
-GPG keys used by the Fedora Project can be found at
-https://fedoraproject.org/keys
---------------------------------------------------------------------------------
-"""
-
-maillist_template = u"""\
-================================================================================
- %(name)s-%(version)s-%(release)s (%(updateid)s)
- %(summary)s
---------------------------------------------------------------------------------
-%(notes)s%(changelog)s%(references)s
-"""
+
+def read_template(name):
+    """
+    Read template text from file.
+
+    Args:
+        name (basestring): The name of the email template stored in 'release' table in database.
+    Returns:
+        basestring: The text read from the file.
+    """
+    location = config.get('mail.templates_basepath')
+    directory = get_absolute_path(location)
+    file_name = "%s.tpl" % (name)
+    template_path = os.path.join(directory, file_name)
+
+    if os.path.exists(template_path):
+        try:
+            with open(template_path) as template_file:
+                return to_unicode(template_file.read())
+        except IOError as e:
+            log.error("Unable to read template file: %s" % (template_path))
+            log.error("IO Error[%s]: %s" % (e.errno, e.strerror))
+    else:
+        log.error("Invalid template name: %s." % (name))
```

Comment:
I wondering if we really need to log this error , I think the `Path does not exist: ...` is enough